KIC: Fix GRPC Guide (#6473)
Kong/docs.konghq.com: app/_includes/md/kic/add-certificate.md

{% unless include.cert_required %}

> The routing configuration can include a certificate to present when clients connect
> over HTTPS. This is not required, as {{site.base_gateway}} will serve a default
> certificate if it cannot find another, but including TLS configuration along
> with routing configuration is typical.

Reviewer: ```suggestion
The routing configuration can include a certificate to present when clients connect
over HTTPS. This is not required, as {{site.base_gateway}} serves a default
certificate if it cannot find another. However, typically you need to include TLS configuration along with routing configuration.
```
